### PR TITLE
Landing page with CDN, Reorg menus, Edit on Github

### PR DESCRIPTION
**Pull request by jeffpatzer**
- Adds two options to the landing page
- One for the existing Getting started flow 
- Another for the new, trimmed down "traditional" flow
- Reorgs menus. Adds in @markbrocato and @ianand recorded convo feedback
- Edit on Github

image

OLD
image

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -44,6 +44,14 @@ const EXAMPLES_REPOS = {
   'static-vue': 'https://github.com/layer0-docs/static-vuejs-example',
 }
 
+const SYSTEM_REQUIREMENTS = `## System requirements
+
+- [Install Node.js 14.0](/guides/install-nodejs) or later`
```
Comment (tristanlee85) [inline constants.js:49]:
```suggestion
- [Install Node.js 14.0](/guides/install-nodejs)`
```

File: constants.js [hunk 2]
```diff
@@ -44,6 +44,14 @@ const EXAMPLES_REPOS = {
...
+
+const SYSTEM_REQUIREMENTS_H3 = `### System Requirements
+
+- [Install Node.js 14.0](/guides/install-nodejs) or later`
+
```
Comment (tristanlee85) [inline constants.js:54]:
```suggestion
const SYSTEM_REQUIREMENTS = `## System requirements

- [Install Node.js 14.0](/guides/install-nodejs)`

const SYSTEM_REQUIREMENTS_H3 = `### System Requirements

- [Install Node.js 14.0](/guides/install-nodejs)`

```

File: guides/install-nodejs.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Install Node.js and npm
+
+**{{ PRODUCT_NAME }} only supports Node.js version 14 and higher**
```
Comment (tristanlee85) [inline guides/install-nodejs.md:3]:
```suggestion
**{{ PRODUCT_NAME }} only supports Node.js version 14**
```

**Comment by ianand** [comment]
- Overall really like the progress. This is great.
- Like the Estimated time to completion in the tutorial. Nice touch.

Still to address before release:

- I'll need to update the terminology and positioning on the front page before we push it live. In fact, I need to a pass on the entire onboarding content.
- The onboarding flow needs to be updated to reflect the new create project interface in the console that @tristanlee85 and team are working on.
- The thing that's still missing, is the bottom area that lists the frameworks is still there even if you select the non-jamstack flow. We still need something in between that indicates it's for jamstack sites.

**Comment by jeffpatzer** [comment]
@ianand let's get this done today. Let's not have you be the bottle neck if we can avoid it
* What language needs to change? 
* I believe the onboarding flow is mostly updated with the two page options. What more do we need to get done for the first rev? 
* I don't follow on your last point. You are saying its still there but also still needed? Please clarify.

**Comment by jeffpatzer** [comment]
@ianand where are you with your copy changes?

**Comment by ianand** [comment]
@jeffpatzer still to do but plan is to get to it this week

**Comment by jeffpatzer** [comment]
@ianand any update here?

**Pull request merged**